### ISSUE TITLE
Is nmapsi4.org still an official domain for this project?

### ISSUE THREAD

**Issue by Gitoffthelawn** [nmapsi4/nmapsi4#6]
Is nmapsi4.org still an official domain for this project?  The site now contains several sections that look like spam, with spam links.

If it's not an official site, perhaps you can notify downstream repositories such as the one used by Linux Mint's Software Manager?  It is still referencing that website.

Related: https://github.com/nmapsi4/nmapsi4/issues/3

**Comment by fcecconi** [comment]
No, it's no longer the project's domain. The official page is just this GitHub repository.

**Comment by Gitoffthelawn** [comment]
Thanks.  Has a spammer taken it over then?

Would it be possible to notify downstream repos that are still sending people to that domain?

**Comment by fcecconi** [comment]
It is necessary to open a bug report (for Debian-based distributions it will be enough to do it on debian bug tracking...) to request the change from the old website to the new one on GitHub.

**Comment by Gitoffthelawn** [comment]
So are spammers now controlling the nmapsi4.org domain?

Is filing a bug report something you are comfortable doing?  I feel as the author, you are the best person to do so.
